package.json

```diff
@@ -73,13 +73,14 @@
     "nodemailer": "^2.0.0",
     "rc": "^1.1.6",
     "readable-stream": "^2.0.5",
+    "redis": "^2.7.1",
     "storj-analytics": "^1.0.0",
     "storj-complex": "^5.1.0",
     "storj-lib": "^6.2.0",
     "storj-mongodb-adapter": "^7.0.1",
     "storj-service-error-types": "^1.1.0",
```

Comment:
This is automatically being upgraded, but it could be good to include `1.2.0` specifically for storj-service-error-types.